libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -12,6 +12,7 @@
 #include <AC_PID/AC_PID_2D.h>       // PID library (2-axis)
 #include <AP_InertialNav/AP_InertialNav.h>  // Inertial Navigation library
 #include "AC_AttitudeControl.h"     // Attitude control library
+#include <AP_InertialSensor/AP_InertialSensor.h>
```

Comment:
No big deal but I wonder if we really need to include AP_InertialSensor.h.  Certainly we need it in the .cpp but I don't think we need it here in the .h.